### DISCUSSION TITLE
How to Create LXC User & Password After COntainer is Created

### DISCUSSION THREAD

**Discussion by Photo64** [tteck/Proxmox#1880] [Q&A] [answered]
I created a docker lxc using the script and loaded some applications, I now realized I need to ssh into the container however when the container was created there was no user/password option so none was set up. Is there a way to create one now after the creation so that I can ssh into the container?

I saw this discussion however it appears they just recreated the container using the advanced option to set the user/password -> Docker Container Password #1112

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/385

**Comment by lucas-0liveira** [comment]
#385
